gatsby-config.js

```diff
@@ -669,34 +651,49 @@ module.exports = {
                             }
                         ]
                     },
-                    {
-                        title: "Frequently asked questions",
-                        path: "documentation/faq"
-                    },
                     {
                         title: "Privacy and GDPR",
-                        path: "documentation/privacy-and-gdpr"
-                    },
-                    {
-                        title: "Manage Gradle dependencies",
-                        path: "documentation/manage-gradle-dependencies"
+                        path: "documentation/resources/privacy-and-gdpr"
                     },
                     {
-                        title: "Manage SPM dependencies",
-                        path: "documentation/manage-spm-dependencies"
+                        title: "Frequently asked questions",
+                        path: "documentation/resources/faq"
                     },
                     {
                         title: "Version 4 Mobile SDKs end-of-support FAQ",
-                        path: "/documentation/v4-end-of-life-faq"
+                        path: "documentation/resources/v4-end-of-life-faq"
                     },
                     {
-                        title: "Adobe Experience Cloud & Apple's IDFA, Privacy announcement",
-                        path: "/documentation/privacy-announcement"
+                        title: "Upgrade to the Experience Platform SDKs",
+                        path: "documentation/resources/upgrade-platform-sdks",
+                        pages: [
```

Comment:
for the v4 materials I plan to group these two sections together in a separate PR